tests/src/PatternAssertion/InPageNavigationAssert.php

```diff
@@ -31,8 +41,15 @@ protected function getAssertions(string $variant): array {
    */
   protected function assertBaseElements(string $html, string $variant): void {
     $crawler = new Crawler($html);
+
     $inpage_nav = $crawler->filter('ul.nav-pills');
     self::assertCount(1, $inpage_nav);
+
+    $svg_icon = $crawler->filter('h2.bcl-heading svg');
+    self::assertCount(1, $svg_icon);
+
+    $base_element = $crawler->filter('.bcl-inpage-navigation');
+    self::assertStringNotContainsString('d-none', $base_element->attr('class'));
```

Comment:
think the negative asset is not really needed just because we removed this utlitity cass

Reply:
miss tests for the ID at least, possibly icon, and the non-presence of the d-none class.

Fra added this point in the first review..